tests/pebble.rs

```diff
@@ -88,6 +88,48 @@ async fn tls_alpn_01() -> Result<(), Box<dyn StdError>> {
         .await
 }
 
+/// Test subproblem handling by trying to issue for a forbidden identifier.
+#[tokio::test]
+#[ignore]
+async fn forbidden_identifier() -> Result<(), Box<dyn StdError>> {
+    let _ = tracing_subscriber::registry()
+        .with(fmt::layer())
+        .with(EnvFilter::from_default_env())
+        .try_init();
+
+    debug!("starting Pebble CA environment");
+    let config = EnvironmentConfig::default();
+    let forbidden_name = config.pebble.domain_blocklist.first().unwrap();
+    let err_result = Environment::new(EnvironmentConfig::default())
```

Comment:
Nit: `err_result` seems like a slightly misleading name for something that is `Box<dyn StdError>`? I'd just call it `error` (or maybe even `err`).

Reply:
Done.